otel-extension/src/main/scala/io/scalac/mesmer/otelextension/instrumentations/zio/ZIOExecutorMetrics.scala

```diff
@@ -0,0 +1,74 @@
+package io.scalac.mesmer.otelextension.instrumentations.zio
+
+import java.util.function.Consumer
+
+import io.opentelemetry.api.GlobalOpenTelemetry
+import io.opentelemetry.api.common.Attributes
+import io.opentelemetry.api.metrics.Meter
+import io.opentelemetry.api.metrics.ObservableLongGauge
+import io.opentelemetry.api.metrics.ObservableLongMeasurement
+import zio.Executor
+import zio.Unsafe
+import zio.internal.ExecutionMetrics
+
+object ZIOExecutorMetrics {
+
+  private val meter: Meter = GlobalOpenTelemetry.getMeter("mesmer")
+
+  private val executionMetricName: String => String = (suffix: String) => s"mesmer_zio_executor_$suffix"
+
+  def registerExecutorMetrics(executor: Executor): Unit =
+    Unsafe.unsafe { implicit u =>
+      val executorAttributes = Attributes.builder().put("executor", executor.toString).build()
```

Comment:
I think that we should try to distinguish between default `ZScheduler` and `zio.internal.Blocking.blockingExecutor` that should be consistent across multiple run of the application - right now if a service is restarted that run with default `Executor` `exector` attribute will change

Reply:
Good point. Do you think config parameters will be a good idea?

Reply:
But, the question is how do we distinguish the blocking/non-blocking executor in the advice?